ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Block/SupportsPeras.hs

```diff
@@ -66,3 +70,14 @@ instance StandardHash blk => BlockSupportsPeras blk where
 
   perasCertRound = pcCertRound
   perasCertBoostedBlock = pcCertBoostedBlock
+
+instance Serialise (HeaderHash blk) => Serialise (PerasCert blk) where
+  encode PerasCert{..} =
+    encodeListLen 2
+      <> encode pcCertRound
+      <> encode pcCertBoostedBlock
+  decode = do
+    decodeListLenOf 2
+    pcCertRound <- decode
+    pcCertBoostedBlock <- decode
+    pure $ PerasCert pcCertRound pcCertBoostedBlock
```

Comment:
Minor: I think it is somewhat preferable to use `NamedFieldPuns` here, to make sure that we are passing the arguments to the constructor in the right order (ie in its current form, the code would also compile if we wrote `pure $ PerasCert pcCertBoostedBlock pcCertRound`).